roles/nvidia-gpu-operator-node-prep/tasks/main.yml

```diff
@@ -1,14 +1,18 @@
 ---
```

Comment:
do we still need this role? you removed the only place it's referenced

Reply:
In theory, no. If someone wants to rollback and use an older version of the GPU Operator, potentially.